backend/database/migrations/create-users.js

```diff
@@ -1,36 +1,72 @@
-'use strict';
 module.exports = {
-  up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('users', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
-      },
-      username: {
-        allowNull: false,
-        type: Sequelize.STRING
-      },
-      email: {
-        allowNull: false,
-        type: Sequelize.STRING
-      },
-      password: {
-        allowNull: false,
-        type: Sequelize.STRING
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      },
-      updatedAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      }
-    });
-  },
-  down: (queryInterface, Sequelize) => {
-    return queryInterface.dropTable('users');
-  }
-};
+    up: (queryInterface, Sequelize) =>
+        queryInterface.createTable('users', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER,
+            },
+            email: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            password: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            username: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            nickname: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            profilePic: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            profileBanner: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            bio: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            followers: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            following: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            posts: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            likes: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            shares: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },

```

Comment:
Hey, sorry for changing the plan again but as discussed, we will create a table for followers and we probably won't be needing posts, likes, and shares anymore because posts can be retrieved using the author field in posts and Dylan already created the relations between likes and shares